src/components/Add.css

```diff
@@ -0,0 +1,253 @@
+.Addfield {
+  display: flex;
+  justify-content: center;
+}
+
+.Formfield {
+  background-color: #ffffff;
+  margin-top: 10px;
+  box-shadow: 0px 0px 5px 8px rgba(226, 221, 221, 0.3);
+  width: 260px;
+  border-radius: 15px;
+  overflow-y: scroll;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px;
+  padding-bottom: 40px;
+  margin-top: 20px;
+  height: 100vh;
+  width: 80%;
+}
+
+::placeholder {
+  color: black;
+  opacity: 1;
+  font-size: 1.2em;
+  align-self: center;
+  font-weight: 600;
+}
+
+input[type="text"] {
+  box-shadow: 8px 8px 8px #cbced1, inset -8px -8px 8px #e2ecf8;
+}
+
+.InputClinicName {
+  width: 267px;
+  padding: 3px;
+  margin: 10px;
+  color: black;
+  border: none;
+  border-radius: 15px;
+  background-color: #e2ecf8;
+  font-size: 17px;
+  height: 47px;
+  text-align: center;
+  overflow-wrap: break-word;
+}
+
+.InputPlace {
+  width: 267px;
+  padding: 17px;
+  border-radius: 15px;
+  border: none;
+  background-color: #e2ecf8;
+  height: 47px;
+  text-align: center;
+  margin-top: 10px;
+}
+
+.InsuranceSelect {
+  border: none;
+  border-radius: 15px;
+  background-color: #e2ecf8;
+  box-shadow: 8px 8px 8px #babdc0, inset -8px -8px 8px #e2ecf8;
+  margin-top: 18px;
+  padding: 15px;
+  width: 267px;
+  height: 90px;
+  justify-content: space-between;
```

Comment:
Not too sure you can use this, if .InsuranceSelect isn't a flex/grid. I might be wrong though